goals.md

```diff
@@ -9,31 +9,56 @@ Based on this, we would like to bring the goals to the TODO Group for review and
 
 The goals will be revisited and updated yearly.
 
-## [PROPOSED] 2022
+# 📌[PROPOSED] 2022
 
-### 1 Cultivate growth and adoption of OSPOs
+2022 Strategic Goals have been divided between TODO Steering Committee and LF Staff goals. Please note that some of these goals may be the same for both. 
+
+## Steering Committee SG:
+
+### ➡️ Cultivate growth and adoption of OSPOs
+
+- Make it easier to produce high quality TODO group guides and OSPO Use Cases
+- Host 8 OSPOlogy sessions
+- Encourage more active participation in OSPOlogy
+
+### ➡️ Grow the community
+
+- Cultivate relationships with other communities
+
+### ➡️ Create value for all members
+
+- Run at least 2 surveys to get feedback, >80% satisfaction
+- Improve and refresh TODO Website
+- Increase geographical availability 
+- Assess TODO tooling (slack, mailing lists, github)
+
+
+## LF Staff SG:
+
+### ➡️ Cultivate growth and adoption of OSPOs
 
 - Release at least 6 new OSPO case studies
 - Launch 6 new OSPO guides
-- Host 12 OSPOlogy sessions
+- Host 8 OSPOlogy sessions
 - Build out and run OSPO Survey 2022
 - Create and publish a OSPO Maturity Model
 
-## 2 Grow the membership base
+### ➡️ Grow the membership base
 
 - Grow TODO membership by 20%
-- Cultivate relationships with other OSPO adjacent communities like CHAOSS
-- Grow OSPO Ambassadors by 
 
-## 3 Create value for all members
+### ➡️ Create value for all members
 
+- Improve and refresh TODO Website
 - Run at least 2 surveys to get feedback, >80% satisfaction
```

Comment:
Same comment as above to distinguish this from the OSPO survey.

Reply:
Done!